go.mod

```diff
@@ -18,6 +18,8 @@ require (
 	github.com/posener/complete v1.2.3 // indirect
 	github.com/ulikunitz/xz v0.5.7 // indirect
 	github.com/vmihailenco/msgpack v4.0.4+incompatible // indirect
-	github.com/william20111/go-thousandeyes v0.5.3
+	github.com/william20111/go-thousandeyes v0.6
 	golang.org/x/crypto v0.0.0-20200214034016-1d94cc7ab1c6 // indirect
 )
+
+replace github.com/william20111/go-thousandeyes => /Users/joshblan/code/go-thousandeyes
```

Comment:
will need to remove this

Reply:
Apologies.  Despite prior care taken to avoid doing this, I still managed to commit it in the end.  Now fixed.